packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -142,6 +143,16 @@ function extractManifestOutput(output: string): string {
   return stripAnsi(output.substring(start, end + 1));
 }
 
+export function populateMetadata(angularVersion?: string): Metadata {
+  const packageJson = JSON.parse(readFileSync(`${__dirname}/../package.json`, "utf-8"));
```

Comment:
Is this guaranteed to be the case? If this file gets moved around this will break, can we add a check or assert that package.json exists